encode/encode.go

```diff
@@ -1,4 +1,11 @@
-package encoding
+package encode
```

Comment:
since the standard library already has an encoding package I've renamed it to encode